.pytool/Plugin/DscCompleteCheck/DscCompleteCheck.py

```diff
@@ -82,7 +83,7 @@ def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM,
                 try:
                     tc.LogStdOut("Ignoring INF {0}".format(a))
                     INFFiles.remove(a)
-                except:
+                except Exception:

```

Comment:
Is this functionally related to the bug or being fixed as general best practice (avoid catching system exiting exceptions)?

Reply:
This change is not related to the bug, just best practices pointed out by my formatter. My formatter automatically made the except explicitly for `BaseException` and since it was pointed out, I changed it to just `Exception` since excepting `BaseException` is unnecessary and not best practice.

Reply:
That's fine. Just wanted to double check.